packages/bp/src/admin/ui/src/workspace/bots/ImportBotModal.tsx

```diff
@@ -102,18 +107,18 @@ class ImportBotModal extends Component<Props, State> {
 
     this.setState({ filePath: files[0].name })
 
-    if (!this.state.botId.length) {
+    if (!this.state.botId.length || this.state.botId === makeBotId(this.props.currentWorkspace, '')) {
```

Comment:
can't we just check the length of this.state.botName instead of generating the ID of an empty named bot?